providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/resolver/process/targeting/OperatorTest.java

```diff
@@ -33,6 +35,27 @@ void flagKeyPresent() throws TargetingRuleException {
         assertEquals(true, evalVariant);
     }
 
+    @Test
+    void timestampPresent() throws TargetingRuleException {
+        // given
+
+        // rule asserting $flagd.timestamp is a number (i.e., a Unix timestamp)
+        final String targetingRule = "{\"var\":[\"$flagd.timestamp\"]}";
+
+        // when
+        Object evalVariant = OPERATOR.apply("some-key", targetingRule, new ImmutableContext());
+
+        long timestamp = (long)Double.parseDouble(evalVariant.toString());
```

Comment:
```suggestion
        // when
        Object timestampString = OPERATOR.apply("some-key", targetingRule, new ImmutableContext());

        long timestamp = (long)Double.parseDouble(timestampString.toString());
```

The result here isn't really a variant, because of how you wrote the rule. I think this name makes more sense.